Ui/Component/Form/DataProvider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\ProductFAQ\Ui\Component\Form;
+
+use Magento\Ui\DataProvider\AbstractDataProvider;
+
+class DataProvider extends AbstractDataProvider
+{
+    /**
+     * DataProvider constructor.
+     * @param string $name
+     * @param string $primaryFieldName
+     * @param string $requestFieldName
+     * @param \Inchoo\ProductFAQ\Model\ResourceModel\Faq\CollectionFactory $collectionFactory
+     * @param array $meta
+     * @param array $data
+     */
+    public function __construct(
+        string $name,
+        string $primaryFieldName,
+        string $requestFieldName,
+        \Inchoo\ProductFAQ\Model\ResourceModel\Faq\CollectionFactory $collectionFactory,
+        array $meta = [],
+        array $data = []
+    ) {
+        parent::__construct($name, $primaryFieldName, $requestFieldName, $meta, $data);
+
+        $this->collectionFactory = $collectionFactory;
```

Comment:
fali ti property ovaj, nemoj se oslanjat na dinamičku deklaraciju properija

Reply:
prošao sam sve i dodao sve što sam stavio dinamičku deklaraciju